tests/unittests/test_security_install.py

```diff
@@ -172,25 +244,85 @@ def test_policy_install(self):
     @pytest.mark.unit
     def test_policy_group_bind_install(self):
         """Check bind security group to security policy"""
-        self._common_install(
-            "some_id", policy_group_bind.create,
+        self._common_install_read_and_update(
+            "security_group_id|security_policy_id",
+            policy_group_bind.create,
             {'policy_group_bind': {
                 "security_policy_id": "security_policy_id",
                 "security_group_id": "security_group_id"
-            }}
+            }},
```

Comment:
Okay. Looks better, I'm going to run the tests on my side and once they're happy I'll approve.